src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements Stack {
+    private static final int NUMBERS_CAPACITY = 12;
+    private int[] numbers = new int[NUMBERS_CAPACITY];
+
+    private int total = EMPTY_STACK;
+
+    public void push(int in) {
+        if (!isFull()) {
+            numbers[total++] = in;
+        }
+    }
+
+    public boolean isEmpty() {
+        return total == EMPTY_STACK;
+    }
+
+    public boolean isFull() {
+        return total == NUMBERS_CAPACITY;
```

Comment:
Changing EMPTY_STACK value requires such fixes? It does not make sense. Incorrect fix.